administrator/language/de-DE/de-DE.com_joomlaupdate.ini

```diff
@@ -44,16 +44,33 @@ COM_JOOMLAUPDATE_VIEW_COMPLETE_HEADING="Joomla!-Versionsaktualisierungsstatus"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_DESCRIPTION_UPDATE_REQUIRED="Erweiterungen, die mit <span class='label label label-warning'>Ja (X.X.X)</span> gekennzeichnet sind, benötigen möglicherweise ein Update."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_DESCRIPTION_UPDATE_REQUIRED="Erweiterungen, die mit <span class='label label label-warning'>Ja (X.X.X)</span> gekennzeichnet sind, benötigen möglicherweise eine Aktualisierung."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_DIRECTIVE="Funktion"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_DOWNLOAD_IN_PROGRESS="Aktualisierung wird heruntergeladen. Bitte warten ..."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXPLANATION_AND_LINK_TO_DOCS="Die Kompatibilitätsprüfung liefert Informationen über den Server, Einstellungen und installierte Erweiterungen für die Aktualisierung.<br>Weitere Informationen über diese Seite und wie man sich auf die Aktualisierung von Joomla vorbereiten kann, sind in der Dokumentation unter <a class='pre-update-docs' href='https://docs.joomla.org/Pre-Update_Check'>Kompatibilitätsprüfung</a> zu finden."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_RUNNING_PRE_UPDATE_CHECKS="Ausführen von Kompatibilitätsprüfungen"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_RUNNING_PRE_UPDATE_CHECKS_NOTES="Bitte etwas Geduld, während die Kompatibilitätsprüfung der Erweiterungen durchgeführt wird."
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE="Keine Aktualisierung erforderlich"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE_NOTES="<p>Der Entwickler der Erweiterung gibt an, dass die derzeit installierte Version kompatibel ist.</p><p id='updateyellowwarning' class='hidden'>Bitte beachten, ist eine Version mit <span class='label label-warning'>X.X.X</span> gekennzeichnet, wird vom Entwickler für die aktuelle Version von Joomla eine neuere Version der Erweiterung angeboten, als für die neue Version von Joomla. Bitte vor der Aktualisierung den Entwickler kontaktieren.</p>"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE="Aktualisierung erforderlich"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla aktualisiert wird.</p><p id='updateorangewarning' class='hidden'>Bitte besonders vorsichtig sein, wenn die zu aktualisierende Version der Erweiterung nicht auch als kompatibel mit der aktuellen Version von Joomla aufgeführt ist.</p>"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_UPDATE_SERVER_OFFERS_NO_COMPATIBLE_VERSION="Aktualisierungsinformationen nicht verfügbar"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_UPDATE_SERVER_OFFERS_NO_COMPATIBLE_VERSION_NOTES="Erweiterung verwendet nicht das Joomla-Update-System oder der Entwickler hat keine Kompatibilitätsinformationen zur Verfügung gestellt."
```

Comment:
```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_UPDATE_SERVER_OFFERS_NO_COMPATIBLE_VERSION_NOTES="Die Erweiterung(en) verwendet nicht das Joomla-Update-System oder der Entwickler hat keine Kompatibilitätsinformationen zur Verfügung gestellt."
```

oder?

Reply:
Würde das zeilenweise sehen und dann auf "Erweiterung" lassen.

Reply:
"Die Erweiterung verwendet", d.h. das "Die" vorne einfügen reicht aus. Wenn du auch noch den Plural abdecken willst gibt es ein Grammatik-Problem mit "verwendet", und das dann mit Klammern oder Schrägstrichen abbilden zu wollen kann blöd werden. Außerdem wird das doch für jede einzelne Erweiterung angezeigt, oder nicht?

Reply:
Und dann in #1172 nach Umbau auf Update noch mal in einer 3.10 wirklich im Backend anschauen.

Reply:
Wird nicht für jede Erweiterung angezeigt ist der allgemeine Block bei den Erweiterungen.

Reply:
Dann vielleicht besser generell Plural.

Reply:
Dann Plural hier das passende.